src/lib/components/Modal/index.ts

```diff
@@ -1 +1 @@
-export { default as CustomPopup } from "./CustomPopup.svelte";
+export { default as ModalWrapperV1 } from "./ModalWrapperV1.svelte";
```

Comment:
Can we use a more generic name than this? Incase we are planning for a V2, this is fine, else we can keep it is Modal or Popup.

Reply:
resolved